plugins/notifiers/client.go

```diff
@@ -19,7 +20,9 @@ type Client interface {
 }
 
 const (
-	ProviderTypeSlack = "slack"
+	ProviderTypeSlack       = "slack"
+	DefaultRetryCount       = 3
```

Comment:
default value can be handled by `pkg/http` as it's their domain, here in notifier we will just follow their default value a.k.a no need to set anything if these configs are empty